packages/story-editor/src/components/mediaRecording/provider.js

```diff
@@ -233,12 +254,14 @@ function MediaRecordingProvider({ children }) {
   const previousBlobUrl = usePrevious(mediaBlobUrl);
 
   useEffect(() => {
-    if (previousBlobUrl && previousBlobUrl !== mediaBlobUrl) {
+    if (
+      previousBlobUrl &&
+      previousBlobUrl !== mediaBlobUrl &&
+      previousBlobUrl !== originalMediaBlobUrl
+    ) {
```

Comment:
[I'm an idiot](https://github.com/GoogleForCreators/web-stories-wp/pull/12022#issuecomment-1204470892)!

> like it has been garbage collected (or window.URL.revokeObjectURL has been called), but I don't see why that should happen

Maybe because it has been revoked right here? 🤦  Well, I guess this fixes it though